src/main/java/at/hannibal2/skyhanni/data/MayorAPI.kt

```diff
@@ -161,7 +163,11 @@ object MayorAPI {
 
         if (!calendarGuiPattern.matches(event.inventoryName)) return
 
-        val stack: ItemStack = event.inventoryItems.values.firstOrNull { jerryHeadPattern.matches(it.displayName) } ?: return
+        val stack: ItemStack = event.inventoryItems.values.firstOrNull {
+            mayorHeadPattern.matchMatcher(it.displayName) {
+                group("name") == "Jerry"
```

Comment:
why is jerry here?

Reply:
This is for getting the jerry extra mayor
It just directly did a regex check with jerry before, now the regex is updated